README.md

```diff
@@ -11,7 +11,10 @@ For the MIPS official toolchain see
 	$ make CROSS_COMPILE=<toolchain-prefix> pistachio_marduk_defconfig
 	$ make CROSS_COMPILE=<toolchain-prefix>
 
-This will generate u-boot-pistachio-nor.img
+This will generate u-boot-pistachio_<board_name>-<version>.img
+eg. u-boot-pistachio_marduk-2015.10-v1.0.4.img OR
+u-boot-pistachio_<board_name>-<version>-<githash>.img, if there are extra commits above the tag.
+e.g. u-boot-pistachio_marduk-2015.10-v1.0.4-g75cbe38.img
```

Comment:
This doesn't look good on the Readme home page.  Please add proper indentation as follows

This will generate u-boot-pistachio_<board_name>-<version>.img

    eg. u-boot-pistachio_marduk-2015.10-v1.0.4.img

OR u-boot-pistachio_<board_name>-<version>-<githash>.img, if there are extra commits above the tag.

    e.g. u-boot-pistachio_marduk-2015.10-v1.0.4-g75cbe38.img